boards/FSM/Src/App/App_SetPeriodicCanSignals.c

```diff
@@ -17,3 +18,34 @@ void App_SetPeriodicCanSignals_SecondaryFlowMeter(
     App_CanTx_SetPeriodicSignal_SECONDARY_FLOW_RATE(
         can_tx, App_FlowMeter_GetFlowRate(flow_meter));
 }
+
+void App_SetPeriodicCanSignals_LeftWheelSpeedSensor(
+    struct FsmCanTxInterface *can_tx,
+    struct WheelSpeedSensor * wheel_speed_sensor)
+{
+    App_CanTx_SetPeriodicSignal_LEFT_WHEEL_SPEED(
+        can_tx, App_WheelSpeedSensor_GetWheelSpeed(wheel_speed_sensor));
+}
+
+void App_SetPeriodicCanSignals_RightWheelSpeedSensor(
+    struct FsmCanTxInterface *can_tx,
+    struct WheelSpeedSensor * wheel_speed_sensor)
+{
+    App_CanTx_SetPeriodicSignal_RIGHT_WHEEL_SPEED(
+        can_tx, App_WheelSpeedSensor_GetWheelSpeed(wheel_speed_sensor));
+}
+
+void App_SetPeriodicCanSignals_WheelSpeedNonCriticalFault(
```

Comment:
I would remove this entirely and just write code like: https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/582/files#r417060913

Reply:
looks good. removed.